db/migrate/20200411224202_add_in_use_release_to_software.rb

```diff
@@ -0,0 +1,6 @@
+class AddInUseReleaseToSoftware < ActiveRecord::Migration[5.2]
+  def change
+    add_column :softwares, :in_use_release, :string
+    add_column :softwares, :in_use_release_date, :date
```

Comment:
How come the fields from the two proposed migrations aren't represented in the model?

Reply:
`ActiveRecord::Base` doesn't require explicit attribute definitions, it always infers them from the table. When you want validators/call_backs/etc is when you add them to the model explicitly.